src/intTest/resources/fhir/multi_pathology_msg1.fhir.json

```diff
@@ -64,6 +47,50 @@
         ]
       }
     },
+    {
+      "fullUrl": "urn:uuid:314bcfae-c309-4413-abf6-a69cc8c48553",
+      "resource": {
+        "resourceType": "Practitioner",
+        "id": "314bcfae-c309-4413-abf6-a69cc8c48553",
+        "identifier": [
+          {
+            "system": "https://fhir.nhs.uk/Id/sds-user-id",
+            "value": "G3380314"
+          }
+        ],
+        "name": [
+          {
+            "text": "SCOTT"
+          }
+        ]
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:5c94279a-905d-435f-b6dc-149cf790d2cc",
+      "resource": {
+        "resourceType": "Organization",
+        "id": "5c94279a-905d-435f-b6dc-149cf790d2cc",
+        "name": "NORTH DOWN GP"
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:b019bdf7-53fe-42a6-89f4-24a6ab5be2c4",
+      "resource": {
+        "resourceType": "Organization",
+        "id": "b019bdf7-53fe-42a6-89f4-24a6ab5be2c4",
+        "type": [
+          {
+            "coding": [
+              {
+                "code": "dept",
+                "display": "Microbiology"
+              }
+            ]
+          }
+        ],
+        "name": "ST JAMES?'S UNIVERSITY HOSPITAL"
```

Comment:
Should the `?` have been handled by the mapper/parser?

Reply:
Yeah should be handled by the parser. I'll update this

Reply:
`MappingUtils.unescape(edifact)` may help.